modules/update-availability-faq.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
+//
+// * updating/understanding-openshift-updates.adoc
+
+:_content-type: CONCEPT
+[id="update-availability_{context}"]
+= Common questions about update availability
+
+There are several factors that affect if and when an update is made available to an {product-title} cluster. The following list provides common questions regarding the availability of an update:
+
+[id="channel-differences_{context}"]
+*What are the differences between each of the update channels?*
+
+* A new release is initially added to the `candidate` channel and is often available as an update target for clusters on the `candidate` channel.
+
+* After successful internal and external testing, a release on the `candidate` channel is supported by being promoted to the `fast` channel, and is often available to clusters on the `fast` channel.
```

Comment:
```suggestion
* After successful final testing a release on the `candidate` channel is promoted to the `fast` channel, an errata is published, and the release is now fully supported.
```

Reply:
I'm assuming you suggested removing "often available to clusters on the `fast` channel" wording because it is extraneous or should not be the focus of this answer. To be consistent, I think I'll remove similar wording from the lines above and below, but let me know if I should keep those phrases instead.

Reply:
Yeah, sorry I'd typed up comment but failed to save. It seemed too hand wavy to me, if it passes tests it goes out. Sounds good to me to remove that consistently.